pkg/nsx/services/securitypolicy/compare.go

```diff
@@ -89,18 +89,32 @@ func SecurityPolicyToComparable(sp *model.SecurityPolicy) Comparable {
 	return (*SecurityPolicy)(sp)
 }
 
-func RulesToComparable(rules []model.Rule) []Comparable {
+func RulesPtrToComparable(rules []*model.Rule) []Comparable {
```

Comment:
why not change SecurityPolicyToComparable to SecurityPolicyPtrToComparable to make naming convention  consistent with RulesPtrToComparable, GroupsPtrToComparable etc.